thinkhazard_processing/processing.py

```diff
@@ -48,6 +51,87 @@ def process(hazardset_id=None, force=False):
         process_hazardset(hazardset, force=force)
 
 
+def upscale_hazardcategories(target_adminlevel_mnemonic):
+    # identify the admin divisions matching the given level:
+    admindivs = DBSession.query(AdministrativeDivision)\
+        .join(AdminLevelType, AdministrativeDivision.leveltype_id == AdminLevelType.id) \
```

Comment:
You shouldn't have to specify on which the join is to be done. SQLAlchemy will determine it.